packages/cashier-v2/src/lib/Transfer/hooks/useExtendedTransferAccounts.ts

```diff
@@ -0,0 +1,101 @@
+import { useActiveAccount, useCurrencyConfig } from '@deriv/api-v2';
+import { THooks } from '../../../hooks/types';
+
+type TGetCurrencyConfig = ReturnType<typeof useCurrencyConfig>['getConfig'];
+
+const sortedMT5Accounts = (accounts: THooks.TransferAccounts, getConfig: TGetCurrencyConfig) => {
+    return [
+        ...accounts
+            .filter(account => account.account_type === 'mt5')
+            .map(account => ({
+                ...account,
+                currencyConfig: account?.currency ? getConfig(account.currency) : undefined,
+            })),
+    ];
```

Comment:
`map` already returns a new array, we don't need to spread the array here again
```suggestion
    return accounts
            .filter(account => account.account_type === 'mt5')
            .map(account => ({
                ...account,
                currencyConfig: account?.currency ? getConfig(account.currency) : undefined,
            }));
```

Reply:
Oh I missed this 😓 Will clean it up.